### PR TITLE
docs: Reword README

### PR DESCRIPTION
**Pull request by mgaeta**
Updating the copy in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,13 +8,22 @@ Check out [Baton](https://github.com/conductorone/baton) to learn more about the
...
 # Prerequisites
 
-Connector uses [key-pair authentication](https://docs.snowflake.com/en/developer-guide/sql-api/authenticating#using-key-pair-authentication) to access Snowflake API. Generating of key-pair and assigning key-pair to the user is described in [this page](https://docs.snowflake.com/en/user-guide/key-pair-auth). 
+This connector uses 
+[key-pair authentication](https://docs.snowflake.com/en/developer-guide/sql-api/authenticating#using-key-pair-authentication) 
+to access the Snowflake API. The process of generating the key pair and then assigning those keys to a user is described in 
+[the key-pair authentication documentation](https://docs.snowflake.com/en/user-guide/key-pair-auth). 
 
-To the connector you should pass path to the **NOT ENCRYPTED PRIVATE KEY in PEM format** using either `BATON_PRIVATE_KEY_PATH` or `--private-key-path` flag. Along with private key you have to pass public key fingerprint using either `BATON_PUBLIC_KEY_FINGERPRINT` or `--public-key-fingerprint` flag.
+The connector receives the keys in the following ways:
```
Comment (mindymo) [inline README.md:16]: @mgaeta I think a little clarification is needed here: Can the connector authenticate using either of these methods? Is there any reason to choose one over the other?

Comment (mgaeta) [reply README.md]: I just pushed up another commit that I hope clarifies in a visual way.

Comment (mindymo) [reply README.md]: That's terrific, much clearer and seems really helpful for users. Thanks!

**Review by mindymo — APPROVED** [review]

**Comment by mgaeta** [comment]
Hey @ggreer CI is broken, is it ok to merge this as-is or should I try to repair tests while I'm here?

**Pull request merged**